packages/payment-processor/test/payment/erc-20-private-payment-hinkal.test.ts

```
@@ -0,0 +1,149 @@
+    signer: {
+      type: Types.Identity.TYPE.ETHEREUM_ADDRESS,
+      value: payerWallet.address,
+    },
+    paymentNetwork,
+  };
+
+  // Step 5: create & send request
+  const request = await requestClient.createRequest(requestCreateParameters);
+  const requestData = await request.waitForConfirmation();
+  return requestData;
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)